tests/integration/cmd_list_services.go

```diff
@@ -0,0 +1,67 @@
+package integration
```

Comment:
Can you rename this file into `cmd_list_services_test.go`, just to be consistent with how the other files are named?